docs/eck-attributes.asciidoc

```diff
@@ -1,5 +1,5 @@
-:eck_version: 1.3.0
+:eck_version: 1.4.0
 :eck_crd_version: v1
-:eck_release_branch: 1.3
+:eck_release_branch: 1.4
```

Comment:
Should those not be 1.6 too?

Reply:
I think that'll break the documentation on `master`.

Reply:
Yes, it would be better. However, we cannot reference a doc version that doesn't (yet) exist. 

Example, when I set `1.5`, https://elasticsearch-ci.elastic.co/job/elastic+cloud-on-k8s+pull-request+build-docs/138/console failed with: `20:59:22 INFO:build_docs:asciidoctor: WARNING: invalid reference: release-notes-1.5.0`

So, I guess we could set 1.6 if we prepare the files `docs/release-notes/1.6.0.asciidoc` and `docs/release-notes/highlights-1.6.0.asciidoc`.

Reply:
Oh right I forgot about the link checking. 👍